Code-4-Community/lucys-love-bus-frontend
# My Events Page

## Issue

Closes #69

## Changes

- Added a component for the My Events page
- Added My Events tab to navbar (shows up only when user is authenticated)

## Verification Steps

We ran it and ensured that the tab only appears when logged in and that the My Events page looks correct when you click on the tab. We also checked the backend to ensure that the request was received correctly.
